Alignment/CommonAlignmentMonitor/plugins/AlignmentStats.cc

```diff
@@ -163,7 +163,7 @@ void AlignmentStats::analyze(const edm::Event &iEvent, const edm::EventSetup &iS
       AlignmentClusterFlag inval;
 
       bool hitInPixel= (subDet==PixelSubdetector::PixelBarrel)||(subDet==PixelSubdetector::PixelEndcap);
-      bool hitInStrip=(subDet==SiStripDetId::TIB) || (subDet==SiStripDetId::TID) ||(subDet==SiStripDetId::TOB) ||(subDet==SiStripDetId::TEC);
+      bool hitInStrip=(subDet==TrackerTopology::TIB) || (subDet==TrackerTopology::TID) ||(subDet==TrackerTopology::TOB) ||(subDet==TrackerTopology::TEC);
```

Comment:
matter of personal preference, but now that I see it done,  I find it odd to have:
`TrackerTopology::TIB  but `PixelSubdetector::PixelBarrel` in the same file

Reply:
I also defined `TrackerTopology::PXB` and `TrackerTopology::PXF`, with the same values as in `PixelSubdetector` in case (these are 1 and 2, while the strip partitions are 3, 4, 5, and 6), so it would be trivial to change them, but I tried to keep this PR minimal (other opinions on naming etc. are welcome - I don't know how things are called for phase2, while TrackerTopology should work for all detectors).